data/tosca/yorc-slurm-types.yml

```diff
@@ -42,8 +42,8 @@ data_types:
         type: integer
         required: false
       mem_per_node:
-        type: integer
-        description: The memory per node in GB required to the job.
+        type: string
```

Comment:
isn't a [`scalar-unit.size`](http://docs.oasis-open.org/tosca/TOSCA-Simple-Profile-YAML/v1.2/TOSCA-Simple-Profile-YAML-v1.2.html#TYPE_TOSCA_SCALAR_UNIT_SIZE) a better type for this?